### PR TITLE
Optimize user guidance

### PR DESCRIPTION
**Pull request by NilsJacobsen**
We added three guides, not linked yet.

- Build a global Svelte App
- What is inlang?
- General guide

### CODE REVIEW

File: inlang/guides/how-to-setup-inlang-for-your-project/marketplace-manifest.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+	"id": "guide.niklasbuchfink.howToSetupInlang",
```
Comment (samuelstroschein) [inline inlang/guides/how-to-setup-inlang-for-your-project/marketplace-manifest.json:2]: id are not required for guides. we will give content random ids that are immutable. @floriandwt are you progressing with ids for content/products?

Comment (floriandwt) [reply inlang/guides/how-to-setup-inlang-for-your-project/marketplace-manifest.json:2]: @samuelstroschein What do you have in mind for the guide manifests exactly?
The last time we talked, we decided to treat them like items for easier maintainability, and just show them in an extra section and under the route `/g/`. 

If you mean the uniqueIDs, they are already implemented as we have sketched in excalidraw: https://inlang.com/m/2qj2w8pu/app-inlang-cli

Comment (floriandwt) [reply inlang/guides/how-to-setup-inlang-for-your-project/marketplace-manifest.json:2]: But I see, there is a bug. Gonna fix it.

Comment (samuelstroschein) [reply inlang/guides/how-to-setup-inlang-for-your-project/marketplace-manifest.json:2]: A perfect, I meant the ids yes! Great to see this implemented.

**Pull request merged**